src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -124,5 +125,13 @@ Integer getInt(Map<Option, ?> options) {
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
-  // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
+  /**
+   * Tests whether the caller has the given permissions on the specified Organization. Returns the
+   * permissions and their results corresponding to whether or not the user has the permission in
```

Comment:
no longer need " corresponding to whether or not the user has the permission in
   * the same position of input list."

Reply:
done